Sources/Basics/Concurrency/AsyncProcess.swift

```diff
@@ -387,6 +387,20 @@ package final class AsyncProcess {
         self.loggingHandler = loggingHandler ?? AsyncProcess.loggingHandler
     }
 
+    package convenience init(
```

Comment:
What's the purpose of this new initialiser? The only difference I can see is that `arguments` renamed to `args`, why would this be needed for enabling tests?

Reply:
The underlying issue is Swift does not allow to easily convert an array to a variadic.